src/wp-admin/js/updates.js

```diff
@@ -305,6 +305,10 @@
 				$menuItem  = $( '#menu-appearance' );
 				$itemCount = $menuItem.find( '.theme-count' );
 				break;
+
+			default:
+				window.console.error( '"%s" is not white-listed to have its count decremented.', type );
```

Comment:
`window.console.error` should be wrapped by `if ( window.console && window.console.error ) { }`.